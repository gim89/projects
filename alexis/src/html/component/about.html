<div id="about" class="about section__padding">
  <div class="container about__container">
    <img src="img/about-img.jpg" alt="about-img" class="about__img">
    <div class="about__part">
      <article class="about__article">
        <h2 class="about__title">About Us.</h2>
        <p class="about__text">
          Analytics release series A financing launch party interaction design android angel investor. Vesting period social media
          sales conversion launch party rockstar incubator customer social proof graphical user interface technology supply
          chain. Non-disclosure agreement rockstar creative partner network launch party social proof paradigm shift long
          tail influencer.
        </p>
        <p class="about__text">
          Monetization validation success. Seed round channels product management. Customer non-disclosure agreement strategy android
          growth hacking ramen buyer.
        </p>
      </article>
      <div class="about__lines-range">
        <div class="about__line line background__red"></div>
        <div class="about__line line background__green"></div>
        <div class="about__line line background__purple"></div>
        <div class="about__line line background__orange"></div>
      </div>
    </div>
  </div>
</div>