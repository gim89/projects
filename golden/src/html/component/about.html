<section class="about section__padding">
  <div class="container">
    <h2 class="about__title">about us</h2>
    <p class="about__subtitle subtitle__margin">Lorem ipsum dolor sit amet consectetur.</p>
    <div class="about__body">
      <div class="about__item">
        <h3 class="about__caption">July 2010</h3>
        <p class="about__subcaption">Our Humble Beginnings</p>
        <p class="about__text">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Ipsa facilis quam ullam consectetur incidunt. Alias accusantium repudiandae odio nemo nulla, ad, dolorum neque, sed libero velit deleniti? Voluptatem, ex fugiat?</p>
      </div>
      <div class="about__item">
        <img src="img/about-one.jpg" alt="" class="about__img">
      </div>
      <div class="about__item"></div>
      <div class="about__item"></div>
      <div class="about__item">
        <img src="img/about-two.jpg" alt="" class="about__img">
      </div>
      <div class="about__item">
        <h3 class="about__caption">January 2011</h3>
        <p class="about__subcaption">Facing Startup Battles</p>
        <p class="about__text">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Ipsa facilis quam ullam consectetur incidunt. Alias accusantium repudiandae odio nemo nulla, ad, dolorum neque, sed libero velit deleniti? Voluptatem, ex fugiat?</p>
      </div>
      
      
      <div class="about__item">
        <h3 class="about__caption">December 2012</h3>
        <p class="about__subcaption">Enter The Dark Days</p>
        <p class="about__text">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Ipsa facilis quam ullam consectetur incidunt. Alias accusantium repudiandae odio nemo nulla, ad, dolorum neque, sed libero velit deleniti? Voluptatem, ex fugiat?</p>
      </div>
      <div class="about__item">
        <img src="img/about-three.jpg" alt="" class="about__img">
      </div>
      <div class="about__item"></div>
      <div class="about__item"></div>
      <div class="about__item">
        <img src="img/about-four.jpg" alt="" class="about__img">
      </div>
      <div class="about__item">
        <h3 class="about__caption">February 2014</h3>
        <p class="about__subcaption">Our Triumph</p>
        <p class="about__text">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Ipsa facilis quam ullam consectetur incidunt. Alias accusantium repudiandae odio nemo nulla, ad, dolorum neque, sed libero velit deleniti? Voluptatem, ex fugiat?</p>
      </div>
      <div class="about__item"></div>
      <div class="about__item">
        <button class="about__btn btn--hover btn--active">our story continues ...</button>
      </div>
      <div class="about__item"></div>
    </div>
  </div>
</section>